src/libraries/System.IO.FileSystem/tests/File/GetSetTimes_SafeFileHandle_Pathless.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Win32.SafeHandles;
+
+namespace System.IO.Tests
+{
+    public sealed class File_GetSetTimes_SafeFileHandle_Pathless : File_GetSetTimes
```

Comment:
suggestion: inherit `File_GetSetTimes_SafeFileHandle`, make `OpenFileHandle` virtual, and override it here. All the other members can be removed.